README.md

```diff
@@ -4,12 +4,12 @@
 [**Audacity**](https://www.audacityteam.org) is an easy-to-use, multi-track audio editor and recorder for Windows, macOS, GNU/Linux and other operating systems. Audacity is open source software licensed under GPL, version 2 or later.
 
 - **Recording** from any real or virtual audio device that is available to the host system.
-- **Export / Import** a wide range of audio formats, extendible with FFmpeg.
+- **Export / Import** a wide range of audio formats, extendable with FFmpeg.
```

Comment:
Should that be "extensible"?

Reply:
I think they are pretty much synonymous. Extensible is probably better though. I'll change it!

Reply:
Changed!